src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -269,21 +269,19 @@
         };
 
         {% if excluded_ajax_paths is defined %}
-            if (window.fetch && window.fetch.polyfill === undefined) {
-                var oldFetch = window.fetch;
-                window.fetch = function () {
-                    var promise = oldFetch.apply(this, arguments);
```

Comment:
we might want to still redefine `window.fetch()` when `</head>` is not found?

Reply:
@nicolas-grekas So check if the `window.Sfjs_globalFetchStack` is set and if not, redefine it?

Reply:
I think so yes

Reply:
@nicolas-grekas OK, I added the proxying if the global stacks are not found